### ISSUE TITLE
No response for JSON output when no validation errors found

### ISSUE THREAD

**Issue by petenattress** [paypal/AATT#24]
The application fails to respond if you send it some simple HTML source with no accessibility issues, and ask for `json` as the output type.

Example - validating this HTML:

``` html
<html lang="en"><head><title>Hello</title></head><body><h1>Heading</h1></body></html>
```

Using this command:

```bash
curl --data 'source=%3Chtml%20lang%3D%22en%22%3E%3Chead%3E%3Ctitle%3EHello%3C%2Ftitle%3E%3C%2Fhead%3E%3Cbody%3E%3Ch1%3EHeading%3C%2Fh1%3E%3C%2Fbody%3E%3C%2Fhtml%3E&output=json' http://localhost:3000/evaluate
```

No response is returned.

I've made a simple proposed fix here: https://github.com/petenattress/AATT/tree/json-bugfix - I can raise a pull request if you like.

**Comment by mpnkhan** [comment]
Yes please. Instead of repeating JSON.stringify and console.log(done) can you Try remove return statements in both loop?
